v4/integrations/nrecho-v3/README.md

```diff
@@ -1,10 +1,10 @@
-# v3/integrations/nrecho-v3 [![GoDoc](https://godoc.org/github.com/newrelic/go-agent/v3/integrations/nrecho-v3?status.svg)](https://godoc.org/github.com/newrelic/go-agent/v3/integrations/nrecho-v3)
+# v4/integrations/nrecho-v3 [![GoDoc](https://godoc.org/github.com/newrelic/go-agent/v4/integrations/nrecho-v3?status.svg)](https://godoc.org/github.com/newrelic/go-agent/v4/integrations/nrecho-v3)
```

Comment:
Should those changes also be made in `integrations/nrecho-v4/README.md`?

Reply:
Great eye!